cpp/src/codegen/native_function.h

```diff
@@ -41,15 +41,18 @@ class NativeFunction {
   ResultNullableType result_nullable_type() const { return result_nullable_type_; }
   bool param_null_safe() const { return param_null_safe_; }
   bool needs_holder() const { return needs_holder_; }
+  bool is_static() const { return static_helper_function_; }
 
  private:
   NativeFunction(const std::string &base_name, const DataTypeVector &param_types,
                  DataTypePtr ret_type, bool param_null_safe,
                  const ResultNullableType &result_nullable_type,
-                 const std::string &pc_name, bool needs_holder = false)
+                 const std::string &pc_name, bool needs_holder = false,
+                 bool static_helper_function = false)
```

Comment:
can we make this a enum instead ?

NativeFunctionType {
  FUNCTION_IR,
  FUNCTION_WITH_HOLDER,
  FUNCTION_STATIC
}